code/modules/admin/tabs/admin_tab.dm

```diff
@@ -421,6 +421,41 @@
 	remove_verb(src, admin_verbs_hideable)
 	add_verb(src, /client/proc/enable_admin_verbs)
 
+/client/proc/strip_all_in_view()
+	set name = "Strip All"
+	set category = "Admin.InView"
+	set hidden = TRUE
+
+	if(!admin_holder || !(admin_holder.rights & R_MOD))
+		to_chat(src, "Only administrators may use this command.")
+		return
+
+	if(tgui_alert(src, "This will strip ALL mobs within your view range. Are you sure?", "Confirmation", list("Yes", "Cancel")))
+		return
+
+	var/strip_self = tgui_alert(src, "Do you want to strip yourself as well?", "Confirmation", list("Yes", "No") == "Yes")
```

Comment:
This is a runtime (at least if you X - it breaks all future tgui_alerts too); the == "Yes" should not be in parenthesis
![image](https://github.com/cmss13-devs/cmss13/assets/76988376/f4e2c288-bd4c-4c68-9e31-6f47d2420267)
Never seen tgui_alert crash but congratulations!
![image](https://github.com/cmss13-devs/cmss13/assets/76988376/49d4906d-c009-4226-b4f6-9f9af9ef8dd7)
Please be sure to test changes to code.